From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: nblog <503407184@qq.com>
Date: Wed, 31 Dec 2025 10:18:57 +0800
Subject: [PATCH] 8842: Documentation Corrections and Script Updates

fix(gdb.html): correct typo in package directory path.
 fix(CreateExampleGDTArchiveScript.java): unify the MSVC version and remove
 the misspelled SDK version.
---
 .../src/main/help/help/topics/gdb/gdb.html                   | 2 +-
 .../Base/ghidra_scripts/CreateExampleGDTArchiveScript.java   | 5 ++---
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/Ghidra/Debug/Debugger-agent-gdb/src/main/help/help/topics/gdb/gdb.html b/Ghidra/Debug/Debugger-agent-gdb/src/main/help/help/topics/gdb/gdb.html
index 04415f321d..81d737b788 100644
--- a/Ghidra/Debug/Debugger-agent-gdb/src/main/help/help/topics/gdb/gdb.html
+++ b/Ghidra/Debug/Debugger-agent-gdb/src/main/help/help/topics/gdb/gdb.html
@@ -140,7 +140,7 @@ python3 -m pip install --no-index -f Debugger-rmi-trace/pypkg/dist -f Debugger-a
     comprising our plugin for GDB and its dependencies. Copy all of the Python packages from
     <TT>Ghidra/Debug/Debugger-rmi-trace/pypkg/dist/</TT> and
     <TT>Ghidra/Debug/Debugger-agent-gdb/pypkg/dist/</TT> to the remote system. It is easiest to put
-    them all in one directory, e.g., <TT>~/ghidra-pypgk/</TT>. Then install them:</P>
+    them all in one directory, e.g., <TT>~/ghidra-pypkg/</TT>. Then install them:</P>
 
     <UL style="list-style-type: none">
       <LI>
diff --git a/Ghidra/Features/Base/ghidra_scripts/CreateExampleGDTArchiveScript.java b/Ghidra/Features/Base/ghidra_scripts/CreateExampleGDTArchiveScript.java
index 60de9e4c4b..09ad80ee16 100644
--- a/Ghidra/Features/Base/ghidra_scripts/CreateExampleGDTArchiveScript.java
+++ b/Ghidra/Features/Base/ghidra_scripts/CreateExampleGDTArchiveScript.java
@@ -110,11 +110,10 @@ public class CreateExampleGDTArchiveScript extends GhidraScript {
 		};
 		
 		String includeFiles[] = {
-			headerFilePath+"/VC/VS22/10.0.190141.0",
-			headerFilePath+"/VC/VS22/10.0.19041.0/um",
+			headerFilePath+"/VC/VS22/Community/VC/Tools/MSVC/14.29.30133/include",
 			headerFilePath+"/VC/VS22/10.0.19041.0/shared",
 			headerFilePath+"/VC/VS22/10.0.19041.0/ucrt",
-			headerFilePath+"/VC/VS22/Community/VC/Tools/MSVC/14.30.30705/include",			
+			headerFilePath+"/VC/VS22/10.0.19041.0/um",
 		};
 		
 		String args[] = {
-- 
2.45.1

